lib/matplotlib/_fontconfig_pattern.py

```diff
@@ -124,47 +95,30 @@ def parse(self, pattern):
         of key/value pairs useful for initializing a
         `.font_manager.FontProperties` object.
         """
-        props = self._properties = {}
         try:
-            self._parser.parseString(pattern)
+            parse = self._parser.parseString(pattern)
         except ParseException as err:
             # explain becomes a plain method on pyparsing 3 (err.explain(0)).
             raise ValueError("\n" + ParseException.explain(err, 0)) from None
-        self._properties = None
         self._parser.resetCache()
+        props = {}
+        if "families" in parse:
+            props["family"] = [*map(_family_unescape, parse['families'])]
+        if "sizes" in parse:
+            props["size"] = [*parse["sizes"]]
```

Comment:
Code coverage thinks this is not tested, but I see a `size=24` in `test_fontconfig_str`. Is that actually a property? Can we expand the test to cover size as well?

Reply:
Sure.  (Note that `size=...` is already tested just above (`size=12.0`).)